crashsimilarity/utils.py

```diff
@@ -34,3 +23,16 @@ def create_dir(path):
     except OSError as e:
         if e.errno != errno.EEXIST:
             raise e
+
+
+CRASHSIMILARITY_DATA_DIR = '../crashsimilarity_data'
```

Comment:
Nit: this should go in downloader.py too and probably `crashes_dump_file_path` too.